### PR TITLE
Added Persian translation of README.md file

### PR DESCRIPTION
**Pull request by H-MAli**

### CODE REVIEW

**Comment by patricklodder** [comment]
I've marked this as "help wanted". Could a native Persian speaker please have a look? Thanks!

**Comment by patricklodder** [comment]
I've marked this as "help wanted". Could a native Persian speaker please have a look? Thanks!

**Comment by MiladGhanei** [comment]
I have read it and everything was correct.

**Review by patricklodder — APPROVED** [review]
ACK based on @MiladGhanei reviewing.

**Review by patricklodder — APPROVED** [review]
re-ACK because I forgot to change the base branch to 1.14.6-dev 😰

File: README_fa_IR.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+<div align="center">
+
+[![DogecoinBadge](https://img.shields.io/badge/Doge-Coin-yellow.svg)](https://dogecoin.com)
+[![MuchWow](https://img.shields.io/badge/Much-Wow-yellow.svg)](https://dogecoin.com)
+
+</div>
+
+Select language: [EN](./README.md) | [CN](./README_zh_CN.md) | FA
+
+دوج‌کوین (Dogecoin) یک رمزارز مبتنی بر عموم مردم بوده که از یک
```
Comment (AlisterTA) [inline README_fa_IR.md:16]: دوج‌کوین (Dogecoin) یک رمزارز جامعه-محور بوده که از یک

File: README_fa_IR.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+[میم اینترنتی](https://fa.wikipedia.org/wiki/%D9%85%DB%8C%D9%85)
```
Comment (AlisterTA) [inline README_fa_IR.md:17]: This line is redundant and does not exist in the original text

File: README_fa_IR.md (hunk 1 above)
Comment (AlisterTA) [inline README_fa_IR.md:16]: دوج‌کوین (Dogecoin) یک رمزارز جامعه-محور بوده که از

**Comment by patricklodder** [comment]
> The text contains many translation errors and It seems the translation was done by an online translation tool. There is a whole bunch of mistakes and it needs to be rewritten entirely.

I have a proposal open to improve how we manage internationalized documentation at #3459. However that is not a popular subject, so I've thus far received 0 comments. In the meantime, feel free to propose corrections using a pull request.

**Pull request merged**